README.md

```
@@ -1,39 +1,44 @@
 # GrantPicks Monorepo
-Pairwise voting mechanism for Stellar built on Soroban (Stellar & NEAR). Mono repo for front end and smart contracts. 
+
+Pairwise voting mechanism for Stellar built on Soroban (Stellar & NEAR). Mono repo for front end and smart contracts.
 
 # Resources
+
 - Technical Architecture https://potlock.org/grantpicks-stellar-architecture
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)